src/games/strategy/triplea/delegate/GameStepPropertiesHelper.java

```diff
@@ -132,15 +132,15 @@ public static boolean isNonCombatMove(final GameData data, final boolean doNotTh
    * Fire rockets after phase is over. Normally would occur after combat move for WW2v2 and WW2v3, and after noncombat
    * move for WW2v1.
    */
-  static boolean isFireRockets(final GameData data) {
+  static boolean isFireRockets(final GameData data, final boolean moveDelegate) {
```

Comment:
1. violation of SRP
2. difficult to tell from a caller perspective what the boolean parameter does. If you were reading that code, to understand `isFireRockets(getData(), false)`, you need to go find that method to look at the parameter. Let's say there is a series of these, it makes it harder to maintain code. Take the extra time now to make it easier for someone to read and use the APIs you are modifying.

Some quick googling, boolean parameters are a generally recognized code smell:
https://blog.ometer.com/2011/01/20/boolean-parameters-are-wrong/
http://softwareengineering.stackexchange.com/questions/147977/is-it-wrong-to-use-a-boolean-parameter-to-determine-behavior

In this case though it does seem a bit messy passing in a boolean parameter. Whoever uses this function will need to be familiar with the implementation.